qa/statsd/src/cases/15.py

```diff
@@ -119,6 +119,20 @@
 ]
 
 command_to_execute = [
+    'echo "statsd.pmda.settings.duration_aggregation_type"',
+    'echo "statsd.pmda.settings.parser_type"',
+    'echo "statsd.pmda.settings.port"',
+    'echo "statsd.pmda.settings.debug_output_filename"',
+    'echo "statsd.pmda.settings.verbose"',
+    'echo "statsd.pmda.settings.max_unprocessed_packets"',
+    'echo "statsd.pmda.settings.max_udp_packet_size"',
+    'echo "statsd.pmda.time_spent_aggregating"',
+    'echo "statsd.pmda.time_spent_parsing"',
+    'echo "statsd.pmda.metrics_tracked"',
+    'echo "statsd.pmda.aggregated"',
+    'echo "statsd.pmda.dropped"',
+    'echo "statsd.pmda.parsed"',
+    'echo "statsd.pmda.received"',
```

Comment:
@Erbenos do all of the above additions need to be prefixed with 'fetch'?  (this is passed to dbpmda right?)

Reply:
Right, was too fast on the trigger

Reply:
Addressed by 516de22